### ISSUE TITLE
Make available on Google Play store

### ISSUE THREAD

**Issue by leif81** [mozilla/MozStumbler#285]
If this were available in the Google Play app store more people could easily install it and keep it up to date.

**Comment by jpyper** [comment]
The devs could use the Play Store staged testing feature. More info here: https://support.google.com/googleplay/android-developer/answer/3131213?hl=en

**Comment by Djfe** [comment]
Facebook has a google group as well which enables the users to get beta updates over the play store so this is a nice idea which I hope the devs will implement some day as it is very useful

**Comment by Djfe** [comment]
@crankycoder will there be a google group for beta staging or not?
else this issue can be closed ;)

**Comment by crankycoder** [comment]
The stable version is on the Google Play https://play.google.com/store/apps/details?id=org.mozilla.mozstumbler now.  F-Droid releases will also be synchronized with Play store releases.  We just got merged into F-Droid (https://gitlab.com/fdroid/fdroiddata/merge_requests/338) so we will shortly appear over there as well.

Github will continue to be used as a beta channel. We can't manage Google Plus for the beta release cycle as we need to manually approve each and every user.
